### DISCUSSION TITLE
The QSO Detail Window - suggestion

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#151] [Ideas]
Hi @foldynl,
I have looked at your testing_0.18 on Linux.  The adjustment to the matrix under the DXCC TAP is very good.  

I suggest you alter the QSO detail window, including the DXCC Tab also there.    This gives the user an easy access to the DXCC status for the entity the call belongs to. 

73 de LB2EG Richard

**Comment by foldynl** [comment]
I'm still optimizing it, so it's not the final form. But the changes will be only about an optimization of algorithm.

I just have one note. In general, testing branches are used for development. The database model can be changed and the code in them is intended only for "development". It is not a beta version, it is actually an integration version. That's why I don't recommend to use it as "main log". it may happen that QSOs in the database may be corrupted. I'm trying to prevent this from happening, but testing breach goes through a long test in the end of development phase and the final version is already the version in the main branch.

> I suggest you alter the QSO detail window, including the DXCC Tab also there. This gives the user an easy access to the DXCC status for the entity the call belongs to.

I will think about it.

I would like to ask you. When I say "Confirmed QSO". Do you accept also eQSL as a confirmation? From my perspective, "confirmed QSO" is QSO that is confirmed only by paper QSL and LoTW, but I would like to ask you how you see it.

**Reply by rsaue** [reply]
Hi, 

Still using LogHX3 and CQRLOG as main logs on win and linux. But QLog is making good progress to eventually be the no 1 on Linux :) Thus, no problem trying dev. versions.. 

As long as we are talking about DXCC, only paper QSL and LOTW are valid. 

BUT, If a user is into eQSL I cannot se any wrong including an OPTION to show and count also eQSLs.  This is how it is done in several other log programs too. 

73 de LB2EG Richard

**Reply by foldynl** [reply]
> BUT, If a user is into eQSL I cannot se any wrong including an OPTION to show and count also eQSLs. This is how it is done in several other log programs too.

that's a way I don't want to go. I would like to keep the options as small as possible. I don't like applications that need many options for everything.

**Comment by rsaue** [comment]
Ok, as long as you do NOT  include eQSL by default :)

Richard
